### PR TITLE
Adds a small asteroid field vault.

### PR DESCRIPTION
**Pull request by Probe1**
Loot: Diamond pickaxe, mineral/random/clown spawns (phazon, bananium, etc).  

Super basic but I think we have a tendency of including too much WOW LOOK AT THIS big time stuff to vaults and some normalcy, even if it's valuable, is nice.  It's just a buncha rocks.

dreamseeker_2018-02-27_07-35-36

### CODE REVIEW

**Comment by Probe1** [comment]
I'm pointing out here that this is very similar to asteroid.dmm but as I said, I want something more low key.  


image

File: maps/randomvaults/objects.dm [hunk 1]
```diff
@@ -46,6 +46,8 @@
 
 /area/vault/asteroid
 
+/area/vault/asteroidfield
```
Comment (MadmanMartian) [inline maps/randomvaults/objects.dm]: As there's nothing really going on in here, why not use the auto-gen area?

**Comment by Sooose** [comment]
I'll give you your thumbs depending on how deep the lore for the clown is.

File: maps/randomvaults/objects.dm (hunk 1 above)
Comment (Probe1) [reply maps/randomvaults/objects.dm]: Shrug.  Habit?  Fear of some code kraken coming from the inky blackness and eating my map.

**Comment by Probe1** [comment]
You're breaking my balls but ok.  I'll add a diary later when I've wrapped up my secret maze vault.

**Comment by Sooose** [comment]
It was mostly a joke but I won't stop you.

**Comment by Probe1** [comment]
I know that's why I'm _obligated_ to follow through.

**Comment by Probe1** [comment]
@Sooose 

```
/obj/item/weapon/paper/asteroidfield
+	name = "dear diary"
+	info = {"It's all HONKING HONKED.  I left for bananium and it HONKED itself to pieces!  Our planet is HONKED!  What am I going to do?"}
```

**Pull request merged**